src/main/java/core/basesyntax/service/impl/SubtractOperationService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.OperationService;
+
+public class SubtractOperationService implements OperationService {

```

Comment:
As for this homework we recommend creating a separate handler for each operation (4 in total) even though they are pretty much the same. These handlers should be named this way: PurchaseOperationHandler and so on. And implement OperationHandler interface. You can use different naming convention, but please don't name they as 'service'